src/SDK/Common/Configuration/Variables.php

```diff
@@ -140,4 +140,6 @@ interface Variables
     public const OTEL_PHP_INTERNAL_METRICS_ENABLED = 'OTEL_PHP_INTERNAL_METRICS_ENABLED'; //whether the SDK should emit its own metrics
     public const OTEL_PHP_DISABLED_INSTRUMENTATIONS = 'OTEL_PHP_DISABLED_INSTRUMENTATIONS';
     public const OTEL_PHP_EXCLUDED_URLS = 'OTEL_PHP_EXCLUDED_URLS';
+    public const OTEL_PHP_SDK_CONFIG_FILE = 'OTEL_PHP_SDK_CONFIG_FILE';
```

Comment:
```suggestion
    public const OTEL_PHP_SDK_CONFIG_FILE = 'OTEL_EXPERIMENTAL_CONFIG_FILE';
```

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#file-configuration

Reply:
Done, and I've set `sdk-config.yaml` as the default, which the spec encourages.